From e25d9a93812847b4ddc9e883d0cd45b32f8e2f76 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Tue, 17 Mar 2020 16:39:07 +0000
Subject: [PATCH] ARM: dts: bcm2711: Add 32-bit PMU compatibility

The "arm" architecture has no support for the cortex-a72 as such, but
the performance and measurement unit from the cortex-a15 seems to be
compatible.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2711-rpi.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

--- a/arch/arm/boot/dts/bcm2711-rpi.dtsi
+++ b/arch/arm/boot/dts/bcm2711-rpi.dtsi
@@ -12,6 +12,10 @@
 		sd_poll_once = <&emmc2>, "non-removable?";
 	};
 
+	arm-pmu {
+		compatible = "arm,cortex-a72-pmu", "arm,cortex-a15-pmu";
+	};
+
 	v3dbus {
 		compatible = "simple-bus";
 		#address-cells = <1>;
